app/components/primer/alpha/action_menu/action_menu_element.ts

```diff
@@ -12,17 +12,6 @@ type SelectedItem = {
 const validSelectors = ['[role="menuitem"]', '[role="menuitemcheckbox"]', '[role="menuitemradio"]']
 const menuItemSelectors = validSelectors.map(selector => `:not([hidden]) > ${selector}`)
 
-export type ItemActivatedEvent = {
-  item: Element
-  checked: boolean
-}
-
-declare global {
-  interface HTMLElementEventMap {
-    itemActivated: CustomEvent<ItemActivatedEvent>
-  }
-}
-
```

Comment:
This might require some updates in dotcom?

Reply:
I don't think so 🤔 I moved these definitions to a new file called shared_events.ts. They should be rolled up into primer_view_components.d.ts... I think? So at worst we'll have to update some imports, but that should be straightforward.